docs-next/pages/roadmap.mdx

```diff
@@ -1,48 +1,52 @@
 import { Markdown } from '../components/Page';
+import { Status } from '../components/Status';
 import { SubscribeForm } from '../components/SubscribeForm';
 
 # Roadmap
 
-When Keystone 5 was announced in 2018, it was a great leap forward for the project from a back-end
-Node.js CMS platform built with MongoDB, to a modern platform with first class support for GraphQL,
-PostgreSQL, and more.
+In 2018 when Keystone 5 was announced, it was a great leap forward from a back-end Node.js CMS platform built with MongoDB, to a modern platform with first class support for GraphQL, PostgreSQL, and more.
 
-Keystone Next is a preview of the next major release of KeystoneJS, the most powerful headless
-content management system around. We've substantially rewritten the CLI, Schema config, and Admin UI
-to make them more powerful and easier to use than ever before.
+Over the last six months, we’ve built around the existing stable core of Keystone 5's packages to test the changes we wanted to make and validate our ideas. Now, in Keystone Next, we've substantially rewritten the CLI, Schema config, and Admin UI to make them more powerful and easier to use than ever before.
 
-In the last six months, we built around the existing stable core of Keystone 5's packages to test
-the changes we wanted to make and validate our ideas. We're now happy with what we've built, and
-over the course of this year, Keystone Next will become Keystone 6.
+There is a lot more to do, and with a larger than ever team at [Thinkmill](https://www.thinkmill.com.au) working on it, we're excited to show you what we're planning and get it shipped!
 
-There is a lot to do, and with a larger than ever team at [Thinkmill](https://www.thinkmill.com.au)
-working on it, we're excited to show you what we're planning and get it shipped!
+We'll be updating this roadmap fortnightly (or as often as we need to) to track progress and plans, find answers or info that matters to your project, and communicate with us more dynamically while we grow the platform.
```

Comment:
This sentence gives my brain a parse error. We start in first person with talk about `We'll be...`, then transition into actions like `find answers` and `communicate with us` that suggest we're now in second person.

Reply:
Totally agree, corrections incoming